packages/map-template/src/atoms/primaryColorState.js

```diff
@@ -0,0 +1,9 @@
+import { atom } from 'recoil';
+
+// The HEX value refers to the --brand-colors-dark-pine-100 from MIDT
+const primaryColorState = atom({
+    key: 'primaryColor',
+    default: '#005655'
```

Comment:
We still don't have a way to use MIDT in JavaScript?
If not, write a comment about which MIDT color this is.

Reply:
No, unfortunately we don't have a way to use MIDT in JS. There is already a comment about which color it refers to. 😁

Reply:
Lol, didn't see it 😄